tests/llama_stack/rag/test_rag.py

```
@@ -29,29 +34,35 @@ class TestLlamaStackRag:
     """
 
     @pytest.mark.smoke
-    def test_rag_inference_embeddings(self, llama_stack_client: LlamaStackClient) -> None:
+    def test_rag_inference_embeddings(
+        self,
+        llama_stack_client: LlamaStackClient,
+        llama_stack_user_config_configmap: ConfigMap,
```

Ruff (0.12.2)

Unused method argument: `llama_stack_user_config_configmap`

(ARG002)